index.js

```diff
@@ -8,40 +8,48 @@ var through2 = require('through2');
 var util = require('util');
 var gutil = require('gulp-util');
 
-module.exports = function (options) {
+var module.exports = function (options) {
+
     options = _.defaults(options || {}, {
         to: null,
         from: null,
         subject: null,
         html: null,
         smtp: null
     });
-    //var transporter = nodemailer.createTransport(
-    //    options.transporter || { service: 'sendmail' }
-    //);
-    var transporter = nodemailer.createTransport('SMTP', options.smtp)
+
+    var transporter = nodemailer.createTransport("SMTP", options.smtp);
```

Comment:
Since the transporter is gonna be closed in each loop, should this declaration be moved into the loop?